rtrlib/rtr_mgr.h

```diff
@@ -67,28 +68,37 @@ struct rtr_mgr_group {
 	enum rtr_mgr_status status;
 };
 
+//TODO Find a nicer way todo a linked list (without writing our own)
+struct rtr_mgr_group_node {
```

Comment:
No, the external API should be minimal and only contain declaration that have a purpose during normal use of the library.
Either the test should use only the external library or we have to find another way to test it. (including the .c file in the test might work)